packages/patternfly-4/react-integration/demo-app-ts/src/Demos.ts

```diff
@@ -50,6 +60,26 @@ export const Demos: DemoInterface[] = [
     name: 'Badge Demo',
     componentType: Examples.BadgeDemo
   },
+  {
+    id: 'blue-bar-grouped-demo',
+    name: 'Bar Chart Grouped Blue Demo',
+    componentType: Examples.BlueBarGroupedDemo
+  },
+  {
+    id: 'color-bar-zoom-demo',
+    name: 'Bar Chart Multi-colored with Zoom Demo',
+    componentType: Examples.ColorBarZoomDemo
+  },
+  {
+    id: 'color-bar-zoom-no-axes-demo',
+    name: 'Bar Chart Multi-colored with Zoom; No X, Y axis Demo',
+    componentType: Examples.ColorBarZoomNoAxesDemo
```

Comment:
We don't need this example. It's been removed it from the react-charts package.

I don't know that we need a one-to-one match of all examples in the demo? I suspect just one full example (e.g., with a legend) would do.

Reply:
Fixed.